init-tools.cmd

```diff
@@ -0,0 +1,79 @@
+@if "%_echo%" neq "on" echo off
```

Comment:
I do agree that using powershell would be better fitting with designs that are currently in the cli repo. We've made changes throughout it's lifetime to use as little cmd as possible.

One reason for this is that the visual and conceptual differences between Bash and powershell is smaller than between bash and cmd. So when you need to maintain dual scripts it makes it that much easier to do so.